docs/source/projects.rst

```diff
@@ -27,6 +27,16 @@ It is pip-installable and published as a PyPI package i.e., you can install it b
 Papers
 *****
 
+FrankenSplit: Efficient Neural Feature Compression With Shallow Variational Bottleneck Injection for Mobile Edge Computing
+----
+* Author(s): Alireza Furutanpey, Philipp Raith, Schahram Dustdar
+* Venue: IEEE Transactions on Mobile Computing
+* PDF: `Paper <https://ieeexplore.ieee.org/document/10480247/>`_
+* Code: `GitHub <https://github.com/rezafuru/FrankenSplitl>`_
```

Comment:
The link is broken. It should be `https://github.com/rezafuru/FrankenSplit`

Reply:
Apologies, should be fine now